mw_api_client/qyoo.py

```diff
@@ -284,7 +286,7 @@ def imageinfo(self, *_, **evil):
         raise NotImplementedError("This module is not implemented due to the \
 corresponding API module's insanity, instability, and pending deprecation.")
 
-    def images(self, limit='max', images=None):
+    def images(self, limit='max', images=None, *_, **evil):
```

Comment:
The function above this one uses `*_` because it's a stub function. For actual API methods, only `**evil` is used.